sourcecode/apis/resources/src/controllers/stats.js

```diff
@@ -12,10 +12,10 @@ export default {
     getResourceStats: async (req) => {
         return {
             data: {
-                last7daysViews:
+                dateRangeViews:
                     await req.context.db.trackingResourceVersion.getCountByDayForResource(
-                        moment().subtract(7, 'days').startOf('day').toDate(),
-                        moment().endOf('day').toDate(),
+                        moment(req.query.start).startOf("day").toDate(),
```

Comment:
Changing from single quotes to double quotes is unnecessary, and introduces inconsistencies with the rest of the file.